### ISSUE TITLE
Bug - Unable to put minecraft in fullscreen

### ISSUE THREAD

**Issue by Rulol4** [leduyquang753/HUDToggler#9]
**Name:** Unable to put minecraft in fullscreen

**Summary:** Cant put minecraft in fullscreen mode, or even reisize the game past a certain size 

**Details:**

Crash message: 
"The game crashed whilst unexpected error
Error: java.lang.ArrayIndexOutOfBoundsException: 3"

How to reproduce:
I installed the mod and joined Hypixel. Everything seemed to be working fine for now. When I tried to put Minecraft in fullscreenmode, it crashed. I relaunched the game. Again, I joined hypixel everything seemed fine. This time I tried manually resizing the game. This seemed to work fine at first, but once I crossed a certain size, the game crashed again with the same error message. I launched the game again, but this time I put the game on my (smaller) second screen. I was able to put it in fullscreen and resize it with no problems; however, my second screen is way smaller than  my first screen, so I usually don't use it for gaming since it is so small. When I tried to move it to my second screen (in full screen), it crashed again. It appears to crash every time the game gets resized to a size larger than the size of my (smaller) second screen, but I am not completely sure that that is the cause. 

[crash-2023-08-04_11.35.21-client.txt](https://github.com/leduyquang753/HUDToggler/files/12259432/crash-2023-08-04_11.35.21-client.txt)

**Comment by leduyquang753** [comment]
Yeah I didn't account for a screen resolution so big that the GUI scaling can be more than 2. This might need a little rework of the scaling selection and I also need to see if I can resetup the development environment, it's been a few years. 😅
